porch/config/deploy/5-rbac.yaml

```diff
@@ -17,16 +17,27 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: aggregated-apiserver-clusterrole
 rules:
-- apiGroups: [""]
-  resources: ["namespaces", "secrets"]
-  verbs: ["get", "watch", "list"]
-- apiGroups: ["admissionregistration.k8s.io"]
-  resources: ["mutatingwebhookconfigurations", "validatingwebhookconfigurations"]
-  verbs: ["get", "watch", "list"]
-- apiGroups: ["config.porch.kpt.dev"]
-  resources: ["repositories"]
-  verbs: ["get", "list", "watch", "create", "update", "patch"]
-# Needed for priority and fairness
-- apiGroups: ["flowcontrol.apiserver.k8s.io"]
-  resources: ["flowschemas", "prioritylevelconfigurations"]
-  verbs: ["get", "watch", "list"]
+  - apiGroups: [""]
```

Comment:
Out of interest - are you using a yaml normalizer / formatter?

Reply:
I think one of the VS Code extension does it for me.